hw/ip/pwrmgr/doc/_index.md

```diff
@@ -119,6 +119,19 @@ If none of these exception cases are matched for low power entry, the fast FSM t
 For reset requests, fall through and aborts are not checked and the system simply resets directly.
 Note in this scenario the slow FSM is not requested to take over.
 
+### ROM Integrity Checks
+
+The power manager coordinates the [start up ROM check]({{< relref "hw/ip/rom_ctrl/doc/_index.md#the-startup-rom-check" >}}) with `rom_ctrl`.
+
+The power manager sends an indication to the `rom_ctrl` to begin performing integrity checks after every reset.
+When the `rom_ctrl` checks are finished, a `done` and `good` indication are sent back to the power manager.
+
+If the device is in life cycle test states (`TEST_UNLOCKED` or `RMA`), the `good` signal is ignored and the ROM contents are always allowed to execute.
+
+If the device is not in one of the test states, the `good` signal is used to determine ROM execution.
+If `good` is true, the ROM execution is allowed.
+If `good` is false, the ROM execution is disallowed.
```

Comment:
I suppose this will lead to some alert or other notification... But the processor cannot run code, so there is no access to CSRs. I am curious how the system deals with this condition.

Reply:
the system doesn't really.. it will just lock-up and hang.
At which point we will have to take it to RMA state and find out what's going on.

So basically...this path better work.

Reply:
:fearful: